src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/service/CertifiedCopyItemService.java

```diff
@@ -62,10 +68,30 @@ public CertifiedCopyItem createCertifiedCopyItem(final CertifiedCopyItem certifi
         } else {
             certifiedCopyItem.getData().setPostalDelivery(Boolean.FALSE);
         }
+        certifiedCopyItem.getData().setKind(KIND);
+        populateItemCosts(certifiedCopyItem, costCalculatorService);
 
         return repository.save(certifiedCopyItem);
     }
 
+    public void populateItemCosts(final CertifiedCopyItem item, final CertifiedCopyCostCalculatorService calculator) {
+        CertifiedCopyItemData itemData = item.getData();
+        String filingHistoryType = itemData.getItemOptions().getFilingHistoryDocuments().get(0).getFilingHistoryType();
```

Comment:
Are we saying then that we will in fact only have a single filing history document to calculate costs for?

Reply:
Ohhh, I missed this. It has to be done for all filing history documents.

Reply:
Thanks Vaughan. I have updated the implementation and tests to handle multiple filing history documents.